chance/p5js/chance_symetry/index.md

```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/chance/p5js/chance_symetry
+title: Symetry
+tags: P5.js, "Law of Connected-Elements", "Law of Regularity", "Law of Symmetry", Chance, Symmetry
```

Comment:
needs to be an array   

    [P5.js, "Law of Connected Elements", "Law of Regularity", "Law of Symmetry", Chance, Symmetry]